CMakeLists.txt

```diff
@@ -126,6 +125,7 @@ else()
   target_link_libraries(rmm INTERFACE CUDA::cudart)
 endif(CUDA_STATIC_RUNTIME)
 
+target_link_libraries(rmm INTERFACE "$<BUILD_INTERFACE:Thrust::Thrust>")
```

Comment:
Why would only the Thrust target go through the `BUILD_INTERFACE` generator but a target like `spdlog::spdlog_header_only` wouldn't?

Reply:
Because this is a hack that still needs to be fixed. This PR is really meant to be WIP for now.

Basically, if something depends on Thrust in this fashion, it can't be installed properly :(

Reply:
I switched it to creating a `rmm::Thrust` target, so the `$<BUILD_INTERFACE...>` is now gone.